examples/node/lib/rest/api/v2010/credentials/aWS.ts

```diff
@@ -426,20 +428,20 @@ export function AWSListInstance(version: V2010): AWSListInstance {
         const data: any = {};
 
         data['TestString'] = params.testString;
-        if (params.testBoolean !== undefined) data['TestBoolean'] = params.testBoolean;
+        if (params.testBoolean !== undefined) data['TestBoolean'] = serialize.bool(params.testBoolean);
         if (params.testInteger !== undefined) data['TestInteger'] = params.testInteger;
         if (params.testNumber !== undefined) data['TestNumber'] = params.testNumber;
         if (params.testNumberFloat !== undefined) data['TestNumberFloat'] = params.testNumberFloat;
         if (params.testNumberDouble !== undefined) data['TestNumberDouble'] = params.testNumberDouble;
         if (params.testNumberInt32 !== undefined) data['TestNumberInt32'] = params.testNumberInt32;
         if (params.testNumberInt64 !== undefined) data['TestNumberInt64'] = params.testNumberInt64;
-        if (params.testObject !== undefined) data['TestObject'] = params.testObject;
-        if (params.testDateTime !== undefined) data['TestDateTime'] = params.testDateTime;
-        if (params.testDate !== undefined) data['TestDate'] = params.testDate;
+        if (params.testObject !== undefined) data['TestObject'] = serialize.object(params.testObject);
+        if (params.testDateTime !== undefined) data['TestDateTime'] = serialize.iso8601DateTime(params.testDateTime);
+        if (params.testDate !== undefined) data['TestDate'] = serialize.iso8601Date(params.testDate);
         if (params.testEnum !== undefined) data['TestEnum'] = params.testEnum;
-        if (params.testObjectArray !== undefined) data['TestObjectArray'] = params.testObjectArray;
+        if (params.testObjectArray !== undefined) data['TestObjectArray'] = serialize.map(params.testObjectArray, function(e) { return e; });
         if (params.testAnyType !== undefined) data['TestAnyType'] = params.testAnyType;
-        if (params.permissions !== undefined) data['Permissions'] = params.permissions;
+        if (params.permissions !== undefined) data['Permissions'] = serialize.map(params.permissions, function(e) { return e; });
```

Comment:
Are there any serde funcions defined which are not used in the current fixture tests?

Reply:
The tests [here](https://github.com/twilio/twilio-node/blob/4.0.0-rc/spec/unit/base/deserialize.spec.js) and [here](https://github.com/twilio/twilio-node/blob/4.0.0-rc/spec/unit/base/serialize.spec.js) seem to catch everything